multiqc/modules/anglerfish/anglerfish.py

```diff
@@ -0,0 +1,247 @@
+                log.debug("No library in general stats table generated from Anglerfish json: {}".format(s_name))
+
+        headers = OrderedDict()
+        headers["library"] = {
+            "namespace": "Anglerfish",
+            "title": "library",
+            "description": "Amount of library used. Calculated from a samples #reads divided by total number of input reads",
+            "max": 100,
+            "min": 0,
+            "scale": "PuBu-rev",
+            "suffix": " %",
+        }
+
+        headers["#reads"] = {
+            "namespace": "Anglerfish",
+            "title": "#reads",
+            "description": "Amount of reads",
+            "min": 0,
+            "scale": "PuOr",
+        }
+        headers["mean_read_len"] = {
+            "namespace": "Anglerfish",
+            "title": "mean_read_len",
+            "description": "Mean read length",
+            "min": 0,
+            "scale": "RdYlGn",
+        }
+        headers["std_read_len"] = {
+            "namespace": "Anglerfish",
+            "title": "std_read_len",
+            "description": "Standard deviation read length",
+            "min": 0,
+            "scale": "RdPu",
+        }
+
+        self.general_stats_addcols(data, headers, "anglerfish")
+
+    def anglerfish_sample_stats(self):
+        """Generate plot for read length from sample stats.
+        For >10 samples: generate table plot
+        for >= 10 samples: generate beeswarm plot"""
+        data = {}
+        debug_list = []
+        total_samples = 0
+        for s_name in self.anglerfish_data:
+            index = self.anglerfish_data[s_name]["sample_stats_amount"]
+            if index > 0:
+                total_samples += index
+                for i in range(index):
+                    sample_name = self.anglerfish_data[s_name]["sample_name_{}".format(i)]
+                    data["Sample: {}".format(sample_name)] = {}
+                    data["Sample: {}".format(sample_name)]["Mean"] = self.anglerfish_data[s_name][
+                        "mean_read_len_{}".format(i)
+                    ]
+                    data["Sample: {}".format(sample_name)]["Standard Deviation"] = self.anglerfish_data[s_name][
+                        "std_read_len_{}".format(i)
+                    ]
+            else:
+                # For non existing sample stat and faulty sample stat
+                debug_list.append(s_name)
+        if len(data) != 0:
+            config = {
+                "id": "Sample_Stat_Read_Length",
+                "title": "Anglerfish: Read Lengths Summary",
+            }
+            # Plot table if less than 10 samples exist, beeswarm if more
+            p = ""
+            if total_samples < 10:
+                p = table.plot(data, None, config)
+            else:
+                p = beeswarm.plot(data, None, config)
+            self.add_section(
+                name="Read Lengths Summary",
+                anchor="Anglerfish-sample-statistics",
```

Comment:
Lower case for HTML anchors generally.

```suggestion
                anchor="anglerfish-sample-statistics",
```

Reply:
Done!